### DISCUSSION TITLE
seo页面重定向是什么意思，怎么使用？

### DISCUSSION THREAD

**Discussion by rabbit2099** [the1812/Bilibili-Evolved#1319] [提问 / Q&A] [answered]
seo页面重定向是什么意思，怎么使用？

**Comment by the1812** [comment, marked as answer]
抱歉描述里写得太潦草了..
一般是谷歌搜索出b站视频的结果时, 会得到一个特殊的专为搜索引擎优化的页面, 称这个为 SEO 页面
这页面也能看视频, 但是视频相关的脚本功能还有夜间模式之类的都会不适配/失效, 所以提供了一个自动跳转到正常视频页面的功能
详情也可以看 https://github.com/the1812/Bilibili-Evolved/issues/649
